client/src/App.jsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/extensions */
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
-import RelatedProducts from './RelatedItems/RelatedProducts.jsx';
+// import RelatedProducts from './RelatedItems/RelatedProducts.jsx';
```

Comment:
Note: In App it looks like the RelatedProducts component was commented out (probably for CSS testing) but not commented back in before PR.